openshift/serverless/operator-install.yaml

```diff
@@ -1,3 +1,17 @@
+# Copyright 2019 The OpenShift Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
Added the license to file as our `./hack/build.sh -x` do license check while building RPM.
Also updated the build_knative_knative function (as seen above) to ensure this is captured in CI and we cut the release branches properly without missing any change. Earlier the license check wasn't running part of CI.